code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -157,6 +159,8 @@
 	display_name = "BCI Core"
 	display_desc = "Controls the core operations of the BCI."
 
+	removable = FALSE
```

Comment:
This isn't necessary. It being a part of the shell *already* makes it non-removable. This just makes it less generic.